os/drivers/audio/alc5658.c

```diff
@@ -688,9 +698,8 @@ static int alc5658_configure(FAR struct audio_lowerhalf_s *dev, FAR void *sessio
 static int alc5658_configure(FAR struct audio_lowerhalf_s *dev, FAR const struct audio_caps_s *caps)
 #endif
 {
-#if !defined(CONFIG_AUDIO_EXCLUDE_VOLUME) || !defined(CONFIG_AUDIO_EXCLUDE_TONE)
 	FAR struct alc5658_dev_s *priv = (FAR struct alc5658_dev_s *)dev;
-#endif
+
```

Comment:
getCaps -> set / configure -> get
you added gain's config in set, but removed all in get.
Aligning needed.
Also if any of CONFIG_AUDIO_EXCLUDE set, return value may OK, not -ENOSYS

Reply:
The configs are recovered.